chronosphere/resource_notification_policy.go

```diff
@@ -320,6 +329,10 @@ func expandNotificationPolicyRaw(
 
 // expandNotificationPolicy converts a notification policy resource to the corresponding API model type.
 func expandNotificationPolicy(p *intschema.NotificationPolicy) (*NotificationPolicyData, error) {
+	isInline := p.Name == ""
+	if p.Slug != "" && isInline {
+		return nil, fmt.Errorf("cannot set slug for unnamed policy, can only set slug for policy with name set")
```

Comment:
nit: not sure why this error message is inconsistent with the one above, should we use the same message in both places?